### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by jparisu**
After PR #305 .

Generated docs: https://eprosima-dds-router.readthedocs.io/en/docs-transparency/

### CODE REVIEW

File: docs/rst/formalia/titlepage.rst [hunk 1]
```diff
@@ -42,7 +42,8 @@ Following are some of the key features of *eProsima DDS Router*:
   6. **Dynamic topic discovery**: the user does not need to fully specify over which topics to communicate (i.e. provide
      concrete topic names and types). The discovery of topics matching the allowlisting rules automatically triggers the
      creation of all entities required for communication.
-
+  7. **Quality of Service preservation**: the DDS Router uses the QoS set by the DDS user network and keeps the
+     reliability and durability of the communication set for each topic. This QoS are also manually configurable.
```
Comment (juanlofer-eprosima) [inline docs/rst/formalia/titlepage.rst:46]:
```suggestion
  7. **Quality of Service preservation**: DDS Router uses the QoS set in the user's DDS network and keeps the
     reliability and durability of the communication for each topic. These QoS are also manually configurable.
```

File: docs/rst/notes/forthcoming_version.rst [hunk 2]
```diff
@@ -6,6 +6,10 @@
 Forthcoming Version
 ###################
 
+Next release will include the following **major features**:
+
+* New **Transparency Module**.
```
Comment (juanlofer-eprosima) [inline docs/rst/notes/forthcoming_version.rst:11]:
```suggestion
* New **QoS Transparency Module**.
```

File: docs/rst/user_manual/configuration.rst [hunk 3]
```diff
@@ -63,23 +79,100 @@ In case this value is not set, the default number of threads used is :code:`12`.
 
 .. _history_depth_configuration:
 
-Maximum History Depth
-=====================
+Maximum Default History Depth
```
Comment (juanlofer-eprosima) [inline docs/rst/user_manual/configuration.rst:82]:
```suggestion
Maximum History Depth
```

File: docs/rst/user_manual/configuration.rst [hunk 4]
```diff
@@ -63,23 +79,100 @@ In case this value is not set, the default number of threads used is :code:`12`.
...
-
 Likewise, one may choose to increase this value if wishing to deliver a greater number of samples to late joiners and
 enough memory is available.
 
 .. _topic_filtering:
 
+Built-in Topics
+===============
+
+|ddsrouter| includes a mechanism to automatically detect which topics are being used in a DDS network.
```
Comment (juanlofer-eprosima) [inline docs/rst/user_manual/configuration.rst:100]: Not sure this is the right definition, I prefer the one below.

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: docs/rst/user_manual/participants/participant.rst [hunk 5]
```diff
@@ -110,6 +110,16 @@ Below is the list with all the available Participant Kinds.
...
           for local communication.
 
+    *   - :ref:`user_manual_participants_discovery_server_wan`
+        - ``wan-discovery-server`` |br|
+          ``wan-ds``
+        - ``guid`` |br|
+          ``listening-addresses`` |br|
+          ``connection-addresses`` |br|
+          ``tls``
+        - Discovery Server DDS DomainParticipant |br|
```
Comment (juanlofer-eprosima) [inline docs/rst/user_manual/participants/participant.rst:120]: (WAN row below) Change description.

File: docs/rst/notes/forthcoming_version.rst [hunk 6]
```diff
@@ -6,6 +6,10 @@
 Forthcoming Version
 ###################
 
+Next release will include the following **major features**:
```
Comment (juanlofer-eprosima) [inline docs/rst/notes/forthcoming_version.rst:9]: (yaml validator below) Supported version v2.

File: docs/rst/user_manual/wan_configuration.rst [hunk 7]
```diff
@@ -243,17 +234,12 @@ User *B* should set its *listening-addresses* and *connection-addresses* as foll
     - name: WANClientParticipant_userB
       kind: wan
 
-      discovery-server-guid:
-        id: 3                             # Must be different than A one
       listening-addresses:
         - ip: 2.2.2.2                     # Public IP of host Hb
           port: 11777                     # Internal and External port
```
Comment (juanlofer-eprosima) [inline docs/rst/user_manual/wan_configuration.rst:239]: (In 3.5.3 Network Address) Say between parenthesis in external port description that it only applies to TCP.

File: docs/rst/user_manual/wan_configuration.rst [hunk 8]
```diff
@@ -101,7 +101,7 @@ TLS
 ===
 
 |eddsrouter| also supports `TLS over TCP <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/tcp/tls.html>`_,
-and its configuration can be set per participant for types Local Discovery Server and WAN. Following is a list of the
+and its configuration can be set per participant for types WAN Discovery Server and WAN. Following is a list of the
```
Comment (juanlofer-eprosima) [inline docs/rst/user_manual/wan_configuration.rst:104]: (Note below about CA)

.. note::
Although in principle only required for TLS clients (with peer verification), the CA (Certification- Authority) file may also be provided
for TLS servers when willing to connect them to other participants configured as servers.

Comment (juanlofer-eprosima) [inline docs/rst/user_manual/wan_configuration.rst:104]: (above in 4.1.1.1 External port)
Write explicitly the tag for external port between parenthesis (`external-port`).

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]
Please, also update the port in last image from WAN communication over TCP use case.

File: docs/rst/notes/forthcoming_version.rst (hunk 6 above)
Comment (jparisu) [reply docs/rst/notes/forthcoming_version.rst:9]: This is not correct. It supportes v3 with some failures

Comment (juanlofer-eprosima) [reply docs/rst/notes/forthcoming_version.rst:9]: To me that means not supported :)

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM

**Pull request merged**